build/templates/examples.rst.mako

```diff
@@ -21,15 +24,15 @@
     from packaging.version import Version
     v = Version(global_version)
     if v.dev is None and v.pre is None:
-        url = 'https://github.com/ni/nimi-python/releases/download/{0}/{1}_examples.zip'.format(global_version, module_name)
+        examples_link_text = '`You can download all {0} examples here <{1}>`_'.format(module_name, released_url)
     else:
-        with open('./LATEST_RELEASE') as vf:
-            latest_release_version = vf.read().strip()
-        url = 'https://github.com/ni/nimi-python/releases/download/{0}/{1}_examples.zip'.format(latest_release_version, module_name)
+        master_url = 'https://github.com/ni/nimi-python/tree/master/src/{}/examples'.format(module_name)
+        examples_link_text = '`You can download all {0} examples for master version here <{1}>`_'.format(module_name, master_url)
```

Comment:
We had agreed to link to the latest release. This is what 90+% of clients will have on their systems since most won't go through the trouble of cloning and building master. And those that do already have the examples locally!

Why this change?

Reply:
Motivation for the change was to fix the broken logic in URL generation. While fixing, I thought we could be explicit and provide links to both `latest release` and `master`.

Reply:
I removed the link to master version of examples, to cater to 90% and avoid confusion